pkg/lang/ir/util.go

```diff
@@ -95,3 +99,14 @@ func (rg *RuntimeGraph) Load(code []byte) error {
 	rg.RuntimeExpose = newrg.RuntimeExpose
 	return nil
 }
+
+func GetDefaultGraphHash() string {
```

Comment:
Could you please add a comment about this func? I cannot get how it does.

Reply:
I just found that It need a little fix because of pointer in the structure, I will add the document When I fix this bug.